### PR TITLE
fastfetch: 2.7.1 -> 2.8.3

### PR DESCRIPTION
**Pull request by iynaix**
## Description of changes

[Changelog](https://github.com/fastfetch-cli/fastfetch/blob/master/CHANGELOG.md)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by khaneliman** [comment]
Result of `nixpkgs-review pr 287671` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages failed to build:</summary>
  <ul>
    <li>fastfetch</li>
    <li>fastfetch.man</li>
  </ul>
</details>

**Comment by khaneliman** [comment]
```bash
> /tmp/nix-build-fastfetch-2.8.2.drv-0/source/src/detection/camera/camera_apple.m:3:9: fatal error: 'AVFoundation/AVCaptureDevice.h' file not found
       > #import <AVFoundation/AVCaptureDevice.h>
       >         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > 1 error generated.
```

File: pkgs/tools/misc/fastfetch/default.nix [hunk 1]
```diff
@@ -95,7 +95,7 @@ stdenv.mkDerivation (finalAttrs: {
     xorg.libXau
     xorg.libXdmcp
     xorg.libXext
-  ] ++ lib.optionals (x11Support && (!stdenv.isDarwin))  [
+  ] ++ lib.optionals (x11Support && (!stdenv.isDarwin)) [
     xfce.xfconf
   ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk_11_0.frameworks; [
     Apple80211
```
Comment (khaneliman) [inline pkgs/tools/misc/fastfetch/default.nix:101]:
```suggestion
    Apple80211
    AppKit
    AVFoundation
```

**Comment by khaneliman** [comment]
Result of `nixpkgs-review pr 287671` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>fastfetch</li>
    <li>fastfetch.man</li>
  </ul>
</details>

**Review by khaneliman — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
@ofborg eval

**Review by SuperSandro2000 — APPROVED** [review]

**Review by khaneliman — APPROVED** [review]

**Comment by iynaix** [comment]
Updated to 2.8.3

**Pull request merged**